wiki/osu-wiki_contribution_guide/en.md

```diff
@@ -0,0 +1,175 @@
+
+![GitHub Preview](7.JPG "Github Preview")
+
+Click on the pencil icon to edit; this should redirect you to the web editor.
+
+![GitHub's Web Editor](8.JPG "web Editor Snapshot")
+
+If you want to preview your changes, click the "Preview Changes" tab. To go back and edit, click the "Edit File" tab.
+
+After making your changes, scroll down and fill the following fields to explain your edit(s) to other contributors.
+
+![Commit Message](9.JPG "Commit Message")
+
+_**Note: Your commit message(s) must thoroughly explain your changes, otherwise, they may not get merged!**_
+
+You would be given an option where to save your edits.
+
+- **"Commit Directly to the ``xyz`` branch"** 
+  - This will commit the changes through your branch directly.
+
+- **"Create a New Branch"** 
+  - Automatically creates a branch for your edits. A commit and a branch with a default name (normally, it would be called ``username : patch-1``) would be created.
+
+A more detailed explanation about this can be found [here]( https://guides.github.com/introduction/flow/).
+
+Pick an option and click "Commit Changes".
+
+### Through your Local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork to your PC and make changes there.
+
+if you have Git installed, enter this in your Command Line Interface (CLI):
+```prolog
+git clone https://github.com/<your github username (ie. peppy)>/osu-wiki 
+```
+*Note: If you don't have Git installed, you can go to [their website](https://git-scm.com/) to download Git and [set it up](https://help.github.com/articles/set-up-git/).*
+
+and this should clone your fork into the folder you specified. You can now proceed to make your changes.
+
+After making your changes locally, you will need to commit this on your local copy.
+
+This can be done by entering this to the Command Line Interface (CLI): 
+```prolog
+git add .
+git commit -m "<explain your changes here>"
+```
+However, some editors (ie. Visual Studio Code) may provide a Git interface.
+
+After committing your changes, you need to push the commits to GitHub so everyone can see it. To do this, enter the following to your Command Line Interface:
+
+```prolog
+git push origin <branch>
+```
+This should prompt you to enter your GitHub login credentials. After you enter them, this should push your commits to GitHub.
+
+_Note: Always check if your copy is synced with your remote. enter ``git status`` to check for any uncommitted changes and the status of your local repositry versus your remote repository._
+
+_Note: However, If your authentication details were saved in Git already, no prompt will appear and will proceed directly to push. This can be done by [caching your Git credentials](https://help.github.com/articles/caching-your-github-password-in-git/)._
+
+## Pull Requests
+
+After pushing your commits, you must make a "Pull Request"(Abbreviated as "PR"). This lets us know that your commits are ready to be reviewed.
```

Comment:
lowercase `a` in the brackets.

`After pushing your commits, you must make a "Pull Request"(abbreviated as "PR"). This lets us know that your commits are ready to be reviewed.`